README.md

```diff
@@ -46,6 +46,8 @@ Note: In `android/app/build.gradle` I tried to set up `compile project(':react-n
 and it compiled well, but the app then crashes after running with initialize/meeting listener.
 So the above solution seems to be the best for now.
 
+Note: If you're using proguard copy the contents of this module's android/proguard.cfg and append them to your project's android/app/proguard-rules.pro otherwise the release build will crash
```

Comment:
could you comment on from where this `android/proguard.cfg` comes from?

Reply:
This file will be ignored in the build process, I've added it here so that people who use proguard can use its contents. I think it'll be better to move it to a separate readme file, your call.
This file is taken from https://github.com/zoom/zoom-sdk-android root directory and I can confirm that without it the proguard enabled release will crash, and although i wish i could shorten this file by removing unnecessary rules, I didn't have the time when making this PR. 
Maybe I'll try this in a future PR.